src/components/PlioQuestion.vue

```diff
@@ -137,6 +150,9 @@ export default {
     // Returns the index of the question that has popped up.
     currentQuestionIndex() {
       return this.plioQuestion.id;
+    },
+    isAnAnsweredQuestion(){
+      return this.plioQuestion.state == "answered"
```

Comment:
A new method to tell if this question has already been answered by the student or not

Reply:
I feel like since it's a one line method, might as well just be an inline condition (Opinion only). Will avoid the overhead of function calls. It's negligible, but still ;-)